src/components/License/License.js

```diff
@@ -109,25 +112,45 @@ class License extends React.Component {
     );
   }
 
-  renderEditLicensePaneMenu = () => (
-    <IfPermission perm="ui-licenses.licenses.edit">
+  renderEditLicensePaneMenu = () => {
```

Comment:
Can you now rename this method to `renderLastMenu` (and update the prop using it), since this name no longer is very accurate? Ie, we're doing more than just rendering the edit button.